<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogStudent</class>
 <widget class="QDialog" name="dialogStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>799</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <widget class="QGroupBox" name="gbStudentEditData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1331</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit data</string>
   </property>
   <widget class="QLabel" name="labelStudentId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Student Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudentName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Full name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStudentId">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStudentName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudentGender">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStudentDateOfBirth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date of birth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStudentCountry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Country:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStCurrentAddress">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEmail">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStProgramCode">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Program name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStRegisteredDate">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Registered date:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditStDateOfBirth">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStCountry">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStAddress">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStEmail">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStAcaProgram">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateStRegisterDate">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbStActive">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Active:</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>280</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QRadioButton" name="rdbtnAddStudent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Student</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnUpdateStudent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Student</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnDeleteStudent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Student</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushBtnStudent">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>320</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dynamic Button</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbGender">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Gender</string>
    </property>
    <widget class="QRadioButton" name="rdStGenderMale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdStGenderFemale">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbActive">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>240</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Active</string>
    </property>
    <widget class="QRadioButton" name="rdStActiveYes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdStActiveNo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbStudentContent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>1331</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Contents</string>
   </property>
   <widget class="QTableView" name="tvStudent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>41</y>
      <width>1291</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
